### PR TITLE
remove all bias

### PR DESCRIPTION
**Pull request by byronbytes**
most importantly there's no more elon

### CODE REVIEW

**Comment by kerolloz** [comment]
LGTM

**Comment by ARemaity** [comment]
Lgtm

**Comment by carloscarrete** [comment]
LGTM

**Comment by RobyCigar** [comment]
LGTM 🤣

**Comment by SaulAryehKohn** [comment]
Looking into it.

**Comment by diguifi** [comment]
duplicate of https://github.com/twitter/the-algorithm/pull/179

**Review by cnnrde — COMMENTED** [review]
LGTM

**Comment by Abastien1734** [comment]
Remove the political variables from the-algorithm/home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/feature_hydrator/RequestQueryFeatureHydrator.scala too

**Comment by arkanttus** [comment]
LGTM

**Comment by datkat21** [comment]
LGTM

**Comment by byronbytes** [comment]
> Remove the political variables from the-algorithm/home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/feature_hydrator/RequestQueryFeatureHydrator.scala too

👍 almost forgot this

**Comment by buddy-codes** [comment]
LGTM.

File: home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala [hunk 1]
```diff
@@ -221,29 +221,6 @@ object HomeTweetTypePredicates {
     ("conversation_module_has_gap", _.getOrElse(ConversationModuleHasGapFeature, false)),
     ("served_in_recap_tweet_candidate_module_injection", _ => false),
     ("served_in_threaded_conversation_module", _ => false),
-    (
-      "author_is_elon",
```
Comment (buddy-codes) [inline home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala:225]: should be author_is_bddylol

Comment (buddy-codes) [reply home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala:225]: then keep that part there

**Review by DaInfLoop — COMMENTED** [review]
lgtm!

File: home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala (hunk 1 above)
Comment (datkat21) [reply home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala:225]: LGTM.

**Comment by EbenezerGH** [comment]
LGTM 👍🏿

**Comment by thedonquixote** [comment]
LGTM

File: home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala [hunk 2]
```diff
@@ -221,29 +221,6 @@ object HomeTweetTypePredicates {
...
-        candidate
-          .getOrElse(AuthorIdFeature, None).contains(candidate.getOrElse(DDGStatsElonFeature, 0L))),
-    (
-      "author_is_power_user",
-      candidate =>
-        candidate
-          .getOrElse(AuthorIdFeature, None)
-          .exists(candidate.getOrElse(DDGStatsVitsFeature, Set.empty[Long]).contains)),
-    (
-      "author_is_democrat",
```
Comment (chaseWillden) [inline home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala:236]: I am surprised this is actually a thing in the code, along with the author_is_republican below.

Comment (byronbytes) [reply home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala:236]: im shocked about this as well, wonder what they're doing with that info 🤔

**Comment by myConsciousness** [comment]
LGTM * 100

**Comment by NicholasDominici** [comment]
LGTM

**Comment by nwithan8** [comment]
LGTM

**Comment by Roniakia** [comment]
lgtm

**Comment by boussouira** [comment]
Looks and feel good :+1:

**Comment by mlemacio** [comment]
lgtm

**Comment by Pridestalkerr** [comment]
lgtm!

File: home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala (hunk 2 above)
Comment (hinell) [reply home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala]: They say it's for metrics only.

Comment (gptlang) [reply home-mixer/server/src/main/scala/com/twitter/home_mixer/functional_component/decorator/HomeTweetTypePredicates.scala]: April fools you fools

**Comment by dubi0us** [comment]
yeaaaaaaaaaaa

**Pull request closed without merging**